cli/util/yaml.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+)
+
+type ProjectConfig struct {
```

Comment:
Why would these not be left in Project? or are you imagining this growing bigger than just project config in the future?  If so, we may want to create a config util as opposed to yaml (or both)